src/Splat/Logging/DebugLogger.cs

```diff
@@ -41,5 +53,11 @@ public void Write(string message, Type type, LogLevel logLevel)
 
             System.Diagnostics.Debug.WriteLine(message, type.Name);
         }
+
+        /// <inheritdoc />
+        public void Write(Exception exception, [Localizable(false)] string message, [Localizable(false)] Type type, LogLevel logLevel)
+        {
+            System.Diagnostics.Debug.WriteLine($"{message} - {exception}", type.Name);
```

Comment:
We should probably add a logLevel check in here as well for this one.

Reply:
apologies missed that. done.